### ISSUE TITLE
AVT 3 - Search Custom Button  component  has an issue navigating with iOS using VO

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4392]
## Environment
iOS 13.1.3 with VoiceOver and Safari

## Description
With VO activated:
1. Swipe Right to the Search component text input field, VO announces "Text field, double-tap to edit, insertion point at end."
2. Enter any text. VO announces the text as its entered. 
3. Swipe R, VO announces, "Clear search input button."
4. Double-tap clears the search input field.
5. Swipe Left. Focus moves to the address bar.
**Expected Result** - Focus should move back to the Search text input field. 

FYI... Screenreader tests with VO and JAWS have no violations.

**Comment by abbeyhrt** [comment]
We have deprecated the Search custom buttons so I'm gonna close this issue!
